[IDE] Remove "Begin completions" and "End completions" from test cases

ahoppen opened the pull request:
These test lines weren't actually providing any value and were annoying to write. Let's jut remove them

ahoppen commented:
@swift-ci Please smoke test

rintaro commented on test/IDE/complete_after_super.swift:
```diff
@@ -397,22 +362,19 @@ class Closures : SuperBaseA {
   func baz() {
     let inner = { [weak self] in
       super.#^CLOSURE_CAPTURE_1?check=CLOSURE_CAPTURE_1;check=NO_SUPER_DECLS;check=NO_CONSTRUCTORS^#
-      // CLOSURE_CAPTURE_1-NOT: Begin completions
```
Or just not to remove `-NOT: Begin completions` for now.

ahoppen commented:
@swift-ci Please smoke test

ahoppen committed:
[IDE] Remove "Begin completions" and "End completions" from test cases

These test lines weren't actually providing any value and were annoying to write. Let's jut remove them.

ahoppen commented:
@swift-ci Please smoke test
